eZ/Publish/Core/FieldType/Tests/RichText/Converter/Xslt/_fixtures/docbook/033-programlisting.xml

```diff
@@ -0,0 +1,20 @@
+<section xmlns="http://docbook.org/ns/docbook" xmlns:ezcustom="http://ez.no/xmlns/ezpublish/docbook/custom" xmlns:ezxhtml="http://ez.no/xmlns/ezpublish/docbook/xhtml" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0-variant ezpublish-1.0">
+    <programlisting><![CDATA[Literal with CNAME end sequence here : ]]]]><![CDATA[> and some more text here]]></programlisting>
+    <programlisting><![CDATA[ordinary literal]]></programlisting>
+    <programlisting><![CDATA[more
+            complicated
+             literal
+              tag]]></programlisting>
+    <para>the php one:</para>
+    <programlisting ezxhtml:class="php"><![CDATA[function foobar()
```

Comment:
At the first glance it looks like a typo and supposedly should be `language`. I know that it doesn't matter, but I suggest using something similar to what you're using below, so e.g. `blue`. Let's think of first-time, inexperienced readers :)

Reply:
The reason why I wrote it like this is that those who haven't created a custom attribute "language" in legacy, likely would have used the class for this purpose instead.
But I'll change it to blue as you suggested

Reply:
Fixed